executor/partition_table_test.go

```diff
@@ -305,12 +330,12 @@ func (s *partitionTableSuite) TestPartitionInfoDisable(c *C) {
 	tk.MustQuery("select * from t_info_null where (date = '2020-10-02' or date = '2020-10-06') and app = 'xxx' and media = '19003006'").Check(testkit.Rows())
 }
 
-func (s *partitionTableSuite) TestOrderByandLimit(c *C) {
-	if israce.RaceEnabled {
-		c.Skip("exhaustive types test, skip race test")
-	}
```

Comment:
Do you know why it is skipped in the past?

Reply:
I don't know. but I have to fix it.

Reply:
Done, All have been removed.